x-pack/plugins/apm/public/components/fleet_integration/apm_policy_form/settings_form/utils.ts

```diff
@@ -60,7 +59,7 @@ export function isSettingsFormValid(
 }
 
 export function validateSettingValue(setting: Setting, value?: any) {
-  if (isEmpty(value)) {
+  if (!isFinite(value) && isEmpty(value)) {
     return {
       isValid: !setting.required,
       message: setting.required ? REQUIRED_FIELD : '',
```

Comment:
Do we show the message if it's valid? If not

```suggestion
      message: REQUIRED_FIELD,
```

Reply:
no, we don't.